lib/solidus_bactracs/api/client.rb

```diff
@@ -21,13 +21,17 @@ def initialize(request_runner:, error_handler:, shipment_serializer:)
         @shipment_serializer = shipment_serializer
       end
 
-      def bulk_create_orders(shipments)
+      def bulk_create_orders(shipments, is_trade_up)
+        if is_trade_up
+          # TODO add implementation for Return Label
```

Comment:
these return labels are something in ShipStation, not Andlor/Bactracs.  Though it's tempting to add it here for convenience, it would entangle / couple the gems and processes in a ways we might argue is risky.  I hope we can find another way